### ISSUE TITLE
R code with smoothing fails on Mac

### ISSUE THREAD

**Issue by devilgate** [smallAreaHealthStatisticsUnit/rapidInquiryFacility#27]
I can now successfully run a submission as long as I check the "Do not apply Bayesian smoothing" option. But if I select one of the other values (at least the first of them, "car_r_procedure: Conditional auto-regressive (CAR) model type"), The R code fails. The relevant error seems to be immediately below, but see the [attached log file](https://github.com/smallAreaHealthStatisticsUnit/rapidInquiryFacility/files/1966905/RIF_middleware.2018-05-02-1.log).

```
callPerformSmoothingActivity() ERROR:  The inla-program exited with an error. Unless you interrupted it yourself, please rerun with verbose=TRUE and check the output carefully.
  If this does not help, please contact the developers at <help@r-inla.org>. ; call stack:  inla.inlaprogram.has.crashed 
callPerformSmoothingActivity exitValue: 1
Closing database connection
Total memory is use: 162428704
Memory by object:
AdjRowset: 51624
connDB: 3632
data: 4945048
errorTrace: 3656
ototal: 56
```

This is on the Mac, so it _may_ be related to not being on Windows, but that seems unlikely. I'm going to leave it alone for now, as I'm sure it can be better explored by an R expert.

**Comment by devilgate** [comment]
Brandon took the batch files and data generated on my Mac, and ran them successfully on Windows. This suggests that the problem is something specific to the Mac. Most likely the version of the INLA R library.

**Comment by devilgate** [comment]
Turns out the solution to this is to not use the version of R installed via HomeBrew, but to download the official package from CRAN, as per [the first comment on this Stack Overflow question](https://stackoverflow.com/questions/45494818/r-library-breaks-due-to-failure-in-loading-dyld). I had to mess around with some settings around dynamic libraries and things, but it's all working now. I'll update the installation instructions accordingly.
